scripts/Util.py

```diff
@@ -738,7 +738,9 @@ def getProps(self, process, current):
                 if self.transport:
                     props["Ice.Default.Transport"] = self.transport
                 if self.protocol:
-                    props["Ice.Default.Protocol"] = self.protocol
+                    useTestHost = isinstance(process.getMapping(current), CSharpMapping) and not process.isFromBinDir()
+                    hostProperty = "Test.Protocol" if useTestHost else "Ice.Default.Protocol"
```

Comment:
Why is the property for protocol derived from useTestHost?

Reply:
This should be `useTestProtocol`